### PR TITLE
Display ruling bonus scores for new Chairman

### PR DESCRIPTION
**Pull request by nwai90**
Closes https://github.com/bafolts/terraforming-mars/issues/2403

image

### CODE REVIEW

File: src/deferredActions/ChoosePoliticalAgenda.ts [hunk 1]
```diff
@@ -17,10 +17,17 @@ export class ChoosePoliticalAgenda implements DeferredAction {
   ) {}
 
   public execute(): PlayerInput {
-    const bonuses: Array<SelectOption> = this.party.bonuses.map((bonus) => new SelectOption(bonus.description, 'Select', () => {
-      this.bonusCb(bonus.id);
-      return undefined;
-    }));
+    const players = this.player.game.getPlayers();
+    const bonuses: Array<SelectOption> = this.party.bonuses.map((bonus) => {
+      let description = bonus.description + ' (';
```
Comment (kberg) [inline src/deferredActions/ChoosePoliticalAgenda.ts:22]: Might as well just make this assigned once as a const.

File: src/turmoil/parties/Greens.ts [hunk 2]
```diff
@@ -33,10 +33,13 @@ class GreensBonus01 implements Bonus {
   id = 'gb01';
   description: string = 'Gain 1 M€ for each Plant, Microbe and Animal tag you have';
 
+  getScore(player: Player) {
+    return player.getTagCount(Tags.PLANT, false, false) + player.getTagCount(Tags.MICROBE, false, false) + player.getTagCount(Tags.ANIMAL, false, false);
```
Comment (kberg) [inline src/turmoil/parties/Greens.ts:37]: You mind making this

```
return player.getTagCount(Tags.PLANT, false, false) +
    player.getTagCount(Tags.MICROBE, false, false) +
    player.getTagCount(Tags.ANIMAL, false, false);
```

for readability?

File: src/turmoil/parties/Greens.ts [hunk 3]
```diff
@@ -46,13 +49,15 @@ class GreensBonus02 implements Bonus {
...
+  }
+
   grant(game: Game) {
     game.getPlayers().forEach((player) => {
-      const count = game.board.spaces.filter((space) => {
-        return space.tile && space.tile.tileType === TileType.GREENERY && space.player !== undefined && space.player.id === player.id;
-      }).length;
-
-      player.addResource(Resources.MEGACREDITS, count * 2);
+      player.addResource(Resources.MEGACREDITS, this.getScore(player));
```
Comment (kberg) [inline src/turmoil/parties/Greens.ts:60]: You think these should be logged? Don't do it now, just asking.

File: src/turmoil/parties/Reds.ts [hunk 4]
```diff
@@ -26,6 +26,19 @@ class RedsBonus01 implements Bonus {
   description = 'The player(s) with the lowest TR gains 1 TR';
   isDefault = true;
 
+  getScore(player: Player) {
```
Comment (kberg) [inline src/turmoil/parties/Reds.ts:29]: Any way to remove the duplication between this and `grant`, below? Same for rb02.

**Review by kberg — APPROVED** [review]

File: src/turmoil/parties/Greens.ts (hunk 3 above)
Comment (nwai90) [reply src/turmoil/parties/Greens.ts:60]: We could. None of these are logged at the moment though

File: src/turmoil/parties/Reds.ts (hunk 4 above)
Comment (nwai90) [reply src/turmoil/parties/Reds.ts:29]: Will look into it. This one is slightly trickier

**Review by kberg — APPROVED** [review]
Nice

**Pull request merged**